services/movie.py

```diff
@@ -0,0 +1,35 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: int = None, actors_ids: int = None) -> None:

```

Comment:
`genres_ids` and `actors_ids` can be None or `list`, so improve type hints, please